lib/mumukit/flow/adaptive_item.rb

```diff
@@ -0,0 +1,57 @@
+module Mumukit::Flow
+  module AdaptiveItem
+    include Mumukit::Flow::Node
+    include Mumukit::Flow::Suggesting
+
+    required :assignment
+    required :children
+    required :parent
+    required :tags
+
+    delegate :passed?, :submitter, to: :@assignment
+
+    def has?(tag)
```

Comment:
Take in mind that `AdaptiveItem` is a mixin, and as such, their methods may collide with those of composed class. 

I'd use a less-generic name, like  `has_tag?` or `tagged_as?`

Reply:
Yes, I liked that it could be read as _has tag?_ but it's too generic.